### ISSUE TITLE
0.25.3 breaking `get_auth_headers` in src/anthropic/lib/bedrock/_auth.py due to bad parameter name

### ISSUE THREAD

**Issue by amiralik16** [anthropics/anthropic-sdk-python#459]
https://github.com/anthropics/anthropic-sdk-python/blob/7312e0a0e296f11f6b6cfee172f182569a5ee849/src/anthropic/lib/bedrock/_auth.py#L44-L49 passes `region_name` to `_get_session` whereas `_get_session` expects `region`. Leading to a :
```
.../lib/python3.10/site-packages/anthropic/lib/bedrock/_auth.py", line 44, in get_auth_headers
    session = _get_session(
TypeError: _get_session() got an unexpected keyword argument 'region_name'
```

The fix should be a one line change to pass the correct argument name.

**Comment by nitinkr0411** [comment]
Facing the same issue as above in 0.25.3

**Comment by RobertCraigie** [comment]
Sorry about this, I had no idea that `functools.lru_cache` discards the function signature...

We'll have a fix out shortly

**Comment by RobertCraigie** [comment]
this has been fixed in 0.25.4
